package/flows/Add_Season.flow-meta.xml

```diff
@@ -0,0 +1,170 @@
+    <label>Add Season</label>
+    <processMetadataValues>
+        <name>BuilderType</name>
+        <value>
+            <stringValue>LightningFlowBuilder</stringValue>
+        </value>
+    </processMetadataValues>
+    <processMetadataValues>
+        <name>CanvasMode</name>
+        <value>
+            <stringValue>AUTO_LAYOUT_CANVAS</stringValue>
+        </value>
+    </processMetadataValues>
+    <processMetadataValues>
+        <name>OriginBuilderType</name>
+        <value>
+            <stringValue>LightningFlowBuilder</stringValue>
+        </value>
+    </processMetadataValues>
+    <processType>Flow</processType>
+    <recordCreates>
+        <name>Create_Season</name>
+        <label>Create Season</label>
+        <locationX>314</locationX>
+        <locationY>518</locationY>
+        <connector>
+            <targetReference>Finish_Season_Screen</targetReference>
+        </connector>
+        <inputAssignments>
+            <field>Sesaon_Name__c</field>
+            <value>
+                <elementReference>Season_Name</elementReference>
+            </value>
+        </inputAssignments>
+        <inputAssignments>
+            <field>TV_Serie__c</field>
+            <value>
+                <elementReference>TvSerieId.Id</elementReference>
+            </value>
+        </inputAssignments>
+        <object>Season__c</object>
+        <storeOutputAutomatically>true</storeOutputAutomatically>
+    </recordCreates>
+    <recordLookups>
+        <name>Get_Season_if_Exists</name>
+        <label>Get Season if Exists</label>
+        <locationX>182</locationX>
+        <locationY>278</locationY>
+        <assignNullValuesIfNoRecordsFound>false</assignNullValuesIfNoRecordsFound>
+        <connector>
+            <targetReference>If_season_already_exists</targetReference>
+        </connector>
+        <filterLogic>and</filterLogic>
+        <filters>
+            <field>Sesaon_Name__c</field>
+            <operator>EqualTo</operator>
+            <value>
+                <elementReference>Season_Name</elementReference>
+            </value>
+        </filters>
+        <filters>
+            <field>TV_Serie__c</field>
+            <operator>EqualTo</operator>
+            <value>
+                <elementReference>TvSerieId.Id</elementReference>
+            </value>
+        </filters>
+        <getFirstRecordOnly>true</getFirstRecordOnly>
+        <object>Season__c</object>
+        <storeOutputAutomatically>true</storeOutputAutomatically>
+    </recordLookups>
+    <screens>
+        <name>Error</name>
+        <label>Error</label>
+        <locationX>50</locationX>
+        <locationY>518</locationY>
+        <allowBack>true</allowBack>
+        <allowFinish>true</allowFinish>
+        <allowPause>false</allowPause>
+        <fields>
+            <name>Display_error_Season</name>
+            <fieldText>&lt;p&gt;The season already exists!&lt;/p&gt;</fieldText>
```

Comment:
Maybe change it into: "The season with the same name(or number) already exist"

Reply:
Done